ci/integration_tests/src/helpers/chain.rs

```diff
@@ -70,12 +67,14 @@ fn global_setup() -> Cfg {
     let gas_report_dir = env::var("GAS_OUT_DIR").unwrap_or_else(|_| "gas_reports".to_string());
 
     let mut cfg = Config::from_yaml(&config).unwrap();
-    let mut orc = CosmOrc::new(cfg.clone())
-        .unwrap()
-        .add_profiler(Box::new(GasProfiler::new()));
+    let mut orc = CosmOrc::new(cfg.clone(), true).unwrap();
 
     let account = test_account(&cfg.chain_cfg.prefix);
 
+    // Poll for first block to make sure the node is up:
+    orc.poll_for_n_blocks(1, Duration::from_millis(20_000), true)
+        .unwrap();
```

Comment:
Locally, If we spin up the juno docker and we have already optimized the contracts then we need to wait a handful of seconds before the first block processes.